modules/roles_profiles/manifests/profiles/disable_services.pp

```diff
@@ -52,6 +52,10 @@
             if $facts['os']['release']['full'] == '10' {
                 include win_disable_services::disable_onedrive
             }
+            ## 2016 == 2022
+            if $facts['os']['release']['full'] == '2016' {
```

Comment:
Does this translate to 2022 or 2016? Would it be better to have a custom fact instead?

Reply:
I am good with leaving it as is for now.  I want to get this out early next week to level 1/2 workers. However, we should refactor to `$facts['custom_win_os_version']` everywhere at some point.